tests/Custom/NodeJS/ServerTest.php

```diff
@@ -47,7 +48,7 @@ protected function getServerScript()
     }
 }
 
-class ServerTest extends \PHPUnit_Framework_TestCase
+class ServerTest extends TestCase
```

Comment:
We're using PHPUnit own base test case class before. Now just use it's namespaced equivalent. No need to use `Behat\Mink\Tests\Driver\Custom\TestCase`, which is causing method declaration incompatibility issues.

Reply:
👍